R/merPredict.R

```diff
@@ -83,7 +93,10 @@ predictInterval <- function(merMod, newdata, which=c("full", "fixed", "random",
                             n.sims = 1000, stat=c("median","mean"),
                             type=c("linear.prediction", "probability"),
                             include.resid.var=TRUE, returnSims = FALSE,
-                            seed=NULL, .parallel = FALSE, .paropts = NULL){
+                            seed=NULL, .parallel = FALSE, .paropts = NULL,
+                            fix.intercept.variance = FALSE, #This does NOT work with random slope models
```

Comment:
I agree with this, let us know when you can do this @jamesonquinn 

Then add yourself as a contributor to the DESCRIPTION